.doc_gen/metadata/bedrock_metadata.yaml

```diff
@@ -48,6 +71,14 @@ bedrock_ListFoundationModels:
             - description: List the available &BRlong; foundation models.
               snippet_tags:
                 - bedrock.java2.list_foundation_models.main
+    JavaScript:
+      versions:
+        - sdk_version: 3
+          github: javascriptv3/example_code/bedrock
+          excerpts:
+            - description: List the available foundation models.
+              snippet_files:
+                - javascriptv3/example_code/bedrock/actions/listFoundationModels.js
```

Comment:
same as the other one, should be `list-foundation-model.js`

Reply:
Same as above